### PR TITLE
Create and Delete GSI sequentially

### PR DESCRIPTION
**Pull request by webwin0**
This pr fixes bug with missed required field for dynamodb `ProvisionedThroughput`
Also, dynamodb doesn't support creating and deleting GSI simultaneously.
So added sequential requests.

### CODE REVIEW

File: lib/Table.js [hunk 1]
```diff
@@ -42,7 +41,9 @@ var compareIndexes = function compareIndexes(local, remote) {
         var remoteIndex = _.pick(remoteIndexes[j], 'IndexName', 'KeySchema', 'Projection');
 
         if (!_.isEqual(remoteIndex, localIndex)) {
-          indexes.both.push(localIndexes[i]);
+          localIndex.ProvisionedThroughput = localIndex.ProvisionedThroughput;
```
Comment (brandongoode) [inline lib/Table.js]: It's early so maybe I'm missing the logic with this line.

**Comment by brandongoode** [comment]
localIndexes[i] should have ProvisionedThroughput but localIndex doesn't.

**Comment by webwin0** [comment]
Fixed :)

**Comment by brandongoode** [comment]
Awesome!  Thanks for the PR.

**Pull request merged**